cmd/utils.go

```diff
@@ -85,6 +85,11 @@ func (handler *SignalHandler) AddCallback(callback SignalCallback) {
 
 // Register should be called as goroutine
 func (handler *SignalHandler) Register() {
+	handler.RegisterWithoutExiting()
+	os.Exit(0)
+}
+
+func (handler *SignalHandler) RegisterWithoutExiting() {
```

Comment:
The linter is going to complain about missing documentation comment.

Reply:
Added missing comment in https://github.com/cossacklabs/acra/pull/393/commits/9194a4c48e5a4d754c0a750034d92487515603db